test/test_other.py

```diff
@@ -15705,7 +15705,8 @@ def test_instantiate_wasm(self):
         });
         return {}; // Compiling asynchronously, no exports.
       }''')
-    self.do_runf('test_manual_wasm_instantiate.c', emcc_args=['--pre-js=pre.js'])
+    # Test with ASYNCIFY here to ensure that that wasmExports gets set to the wrapped version of the wasm exports.
+    self.do_runf(test_file('test_manual_wasm_instantiate.c'),emcc_args=['--pre-js=pre.js','-sASYNCIFY'])
```

Comment:
Space after comms here

And remove the `test_file` function call

Otherwise LGTM now!